docs/README.md

```diff
@@ -101,71 +117,68 @@ module.exports = {
 };
 ```
 
-**"What's changed from A2 in this code?"**
-
-* Project-level modules now reside in `./modules`, not `./lib/modules`.
+### Modules
 
-* Apostrophe's core modules like `@apostrophecms/home-page` are namespaced now, just like newer npm modules. This is true even for modules that ship inside a "bundle" such as the `apostrophe` npm module.
+- Project-level modules have been moved from `./lib/modules` to  `./modules`
+- Apostrophe's core modules like `@apostrophecms/home-page` are namespaced now, just like newer npm modules. This is true even for modules that ship inside a "bundle" such as the `apostrophe` npm module.
 
-* Even the home page has its own module, `@apostrophecms/home-page`. We're just configuring it here, so we don't have to use `extend`. But we will when we add a new page type to the site.
+### Schemas
 
-* Ordinary module options like `label` now go in an `options` property, not at the top level.
+- Ordinary module options like `label` now reside in an `options` property.
+- Every area must be configured as part of the schema for a page-type or piece-type.
+- Fields are added to the module with the new `fields` syntax. New fields are added via the `add` property.
+- You can also `remove` fields by passing an array for `remove`.
+- For convenience, fields are now configured as an object, rather than an array.
 
-* Every area must be configured as part of the schema for a page type or piece type.
```

Comment:
This is a super major A3 change that people need to see.

Reply:
This wasn't removed, it was moved to the shema section breakdown